manifests/k/KDE/KDEConnect/21.08.1.725/KDE.KDEConnect.installer.yaml

```
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: KDE.KDEConnect
+PackageVersion: 21.08.1.725
```

Kinda funny how this is the only manifest we allow to not match ARP #ManifestVersion1.1.0
```suggestion
PackageVersion: 21.08.1
```